libs/core.py

```diff
@@ -66,31 +71,67 @@ def __process(self, cv_image):
             y1 = box[2]
             obj["centroid"] = [(x0 + x1) / 2, (y0 + y1) / 2, x1 - x0, y1 - y0]
             obj["bbox"] = [x0, y0, x1, y1]
-            obj["centroidReal"]=[(x0 + x1)*w / 2, (y0 + y1)*h / 2, (x1 - x0)*w, (y1 - y0)*h]
-            obj["bboxReal"]=[x0*w,y0*h,x1*w,y1*h]
- 
+            obj["centroidReal"] = [(x0 + x1) * w / 2, (y0 + y1) * h / 2, (x1 - x0) * w, (y1 - y0) * h]
+            obj["bboxReal"] = [x0 * w, y0 * h, x1 * w, y1 * h]
+
         objects_list, distancings = self.calculate_distancing(tmp_objects_list)
         return cv_image, objects_list, distancings
 
+    def gstreamer_writer(self, feed_name, fps, resolution):
```

Comment:
It's good to have a docstring here

Reply:
Thanks @emmaWdev . I added the docstring.